README.md

```diff
@@ -22,12 +25,106 @@ It supports TensorFlow, PyTorch, MXNet, and XGBoost on Python 3.6+.
 - Distributed training support
 - TensorBoard support
 
+## Support
+
+### [Latest release v0.7.2](https://github.com/awslabs/sagemaker-debugger/releases)
+
+   - Experimental support for TF 2.x GradientTape - Introducing experimental support for TF 2.x training scripts using GradientTape.
+   SageMaker Debugger GradientTape now captures tensors such as loss, metrics, weights, biases, and gradients by modifying training scripts in TF 2.x framework. A sample training script is provided [here](https://github.com/awslabs/sagemaker-debugger/blob/master/examples/tensorflow2/scripts/tf_keras_gradienttape.py).
+   GradientTape does not work with Zero Script Change experience at this time.
+
+        Note: Training scripts using GradientTape for higher-order gradients or multiple tapes are not supported. Distributed training scripts that use GradientTape are not supported at this time.
+
+   - Support SyncOnReadVariable in mirrored strategy - Fixes a bug that occurred because SyncOnRead distributed variable was not supported with smdebug. Also enables the use of smdebug with training scripts using TF 2.x MirroredStrategy with fit() API.
+
+   - Turn off hook and write only from one worker for unsupported distributed training techniques – Fixes a crash when distributed training in PyTorch framework is implemented using generic multiprocessing library, which is not a method supported by smdebug. This fix handles this case and ensures that tensors are saved.
+
+   - Bug fix: Pytorch: Register only if tensors require gradients – Users were observing a crash when training with pre-trained embeddings which does not need gradient updates. This fix checks if a gradient update is required and registers a backward hook only in those cases.
+
+#### Setting up SageMaker Debugger
+
+- `smdebug` library runs on Python 3.x. Install `smdebug` through:
+
+    ```
+    pip install smdebug
+    ```
+
+#### Zero Script Change
+
+| Framework | Version |
+| --- | --- |
+| [TensorFlow](tensorflow.md) | 1.15, 2.1 |
+| [MXNet](mxnet.md) | 1.6 |
+| [PyTorch](pytorch.md) | 1.3, 1.4 |
+| [XGBoost](xgboost.md) | >=0.90-2 [As Built-in algorithm](xgboost.md#use-xgboost-as-a-built-in-algorithm)|
+
+#### Bring your own training container
+
+| Framework | Versions |
+| --- | --- |
+| [TensorFlow](tensorflow.md) | 1.14. 1.15, 2.0.1, 2.1.0 |
+| Keras (with TensorFlow backend) | 2.3 |
+| [MXNet](mxnet.md) | 1.4, 1.5, 1.6 |
+| [PyTorch](pytorch.md) | 1.2, 1.3, 1.4 |
+| [XGBoost](xgboost.md) | [As Framework](xgboost.md#use-xgboost-as-a-framework) |
+
+#### Support for Distributed Training and Known Limitations
+
```

Comment:
this table should also have ZCC and non-ZCC column.

Reply:
Will open another PR for this and [distributed_train.md](https://github.com/awslabs/sagemaker-debugger/blob/master/docs/distributed_training.md).